lib/crowbar/client/command/backup/download.rb

```diff
@@ -45,21 +43,31 @@ def execute
           protected
 
           def write(body)
-            path.open("wb") do |f|
-              f.binmode
-              f.write body
+            path.binmode
+            path.write body
 
-              true
-            end
+            true
           rescue
             false
           end
 
           def path
-            filename = Request::Backup::List.new.process do |p|
-              p.parsed_response.select { |s| s["id"] == args.id.to_i }
-            end.first["name"]
-            Pathname.new("#{filename}.tar.gz")
+            @path ||= case args.file
+            when "-"
```

Comment:
Style/CaseIndentation: Indent `when` as deep as `case`. (https://github.com/bbatsov/ruby-style-guide#indent-when-to-case)